src/pat/autotoc/autotoc.js

```diff
@@ -107,10 +137,28 @@ export default Base.extend({
                     $(this).trigger("clicked");
                     if (!options.skipHash) {
                         if (window.history && window.history.pushState) {
-                            window.history.pushState({}, "", "#" + $el.attr("id"));
+                            window.history.pushState({}, "", `#${id}`);
                         }
                     }
                 });
+            if (asTabs) {
+                $nav.addClass([
+                    "mt-0",
+                    "me-2",
+                    "ms-2",
+                    "py-2",
+                    "px-3",
+                    "border-bottom-0",
+                ]);
```

Comment:
This is not needed because the Bootstrap `.nav-tabs .nav-link` styles take care of that. Just keep the definitions below with `if (!asTabs)` ...